generated/nifgen/nifgen/_library_singleton.py

```diff
@@ -13,8 +13,8 @@
 _instance = None
 _instance_lock = threading.Lock()
 _library_info = {'Linux': {'64bit': {'name': 'nifgen', 'type': 'cdll'}},
-                 'Windows': {'32bit': {'name': 'nifgen_32.dll', 'type': 'windll'},
-                             '64bit': {'name': 'nifgen_64.dll', 'type': 'cdll'}}}
+                 'Windows': {'32bit': {'name': 'niFgen_32.dll', 'type': 'windll'},
```

Comment:
what's with all this diff noise?

Reply:
Because I updated the nifgen metadata from our internal source and this is what it has.

Reply:
I don't see metadata among the files being changed in this PR.

Reply:
Those metadata updates should have happened in their own separate PRs, no?

Reply:
I'm not sure why. There were multiple changes in the metadata files, and I see the differences in the PR

Reply:
I thought you made them when you updated nifgen.